From 7972c4c29b0cd20a2bac6cc9c40ebca4eb1f6abb Mon Sep 17 00:00:00 2001
From: Felix Radensky <felix.r@variscite.com>
Date: Wed, 15 Jul 2020 17:59:40 +0300
Subject: [PATCH] serial: mxc: Make mxc_base modifiable by external code

Required when several boards use the same board header file
but use different UARTs as console.

(cherry picked from commit b0e838c602e07914317840fc5c80504443d5d3fc)
---
 drivers/serial/serial_mxc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/serial/serial_mxc.c b/drivers/serial/serial_mxc.c
index 058f526b2cc..3c0096ed9f3 100644
--- a/drivers/serial/serial_mxc.c
+++ b/drivers/serial/serial_mxc.c
@@ -185,7 +185,7 @@ static void _mxc_serial_setbrg(struct mxc_uart *base, unsigned long clk,
 #error "define CONFIG_MXC_UART_BASE to use the MXC UART driver"
 #endif
 
-#define mxc_base	((struct mxc_uart *)CONFIG_MXC_UART_BASE)
+struct mxc_uart *mxc_base = ((struct mxc_uart *)CONFIG_MXC_UART_BASE);
 
 static void mxc_serial_setbrg(void)
 {
-- 
2.34.1

